docs/zh_cn/tutorials/0_config.md

```diff
@@ -1,189 +1,187 @@
-    - [Training information](#training-information)
-    - [Data information](#data-information)
-    - [Config File Name Example](#config-file-name-example)
-    - [Checkpoint Naming Convention](#checkpoint-naming-convention)
-  - [Config File Structure](#config-file-structure)
-  - [Inherit and Modify Config File](#inherit-and-modify-config-file)
-    - [Use intermediate variables in configs](#use-intermediate-variables-in-configs)
-    - [Ignore some fields in the base configs](#ignore-some-fields-in-the-base-configs)
-    - [Use some fields in the base configs](#use-some-fields-in-the-base-configs)
-  - [Modify config through script arguments](#modify-config-through-script-arguments)
-  - [Import user-defined modules](#import-user-defined-modules)
+- [教程 0: 学习配置](#教程-0-学习配置)
+  - [配置文件与检查点命名约定](#配置文件与检查点命名约定)
+    - [算法信息](#算法信息)
+    - [模块信息](#模块信息)
+    - [训练信息](#训练信息)
+    - [数据信息](#数据信息)
+    - [配置文件命名示例](#配置文件命名示例)
+    - [检查点命名约定](#检查点命名约定)
+  - [配置文件结构](#配置文件结构)
+  - [继承和修改配置文件](#继承和修改配置文件)
+    - [使用配置中的中间变量](#使用配置中的中间变量)
+    - [忽略基础配置中的字段](#忽略基础配置中的字段)
+    - [使用基础配置中的字段](#使用基础配置中的字段)
+  - [通过脚本参数修改配置](#通过脚本参数修改配置)
+  - [导入用户定义模块](#导入用户定义模块)
 
 <!-- TOC -->
 
-## Config File and Checkpoint Naming Convention
+## 配置文件与检查点命名约定
 
-We follow the below convention to name config files. Contributors are advised to follow the same style. The config file names are divided into four parts: algorithm info, module information, training information and data information. Logically, different parts are concatenated by underscores `'_'`, and words in the same part are concatenated by dashes `'-'`.
+我们遵循下述约定来命名配置文件并建议贡献者也遵循该命名风格。配置文件名字被分成4部分：算法信息、模块信息、训练信息和数据信息。逻辑上，不同部分用下划线连接 `'_'`，同一部分中的单词使用破折线 `'-'` 连接。
 
 ```
 {algorithm}_{module}_{training_info}_{data_info}.py
 ```
 
-- `algorithm info`：Algorithm information includes algorithm name, such as simclr, mocov2, etc.;
-- `module info`： Module information is used to represent some backbone, neck, head information;
-- `training info`：Training information, some training schedule, including batch size, lr schedule, data augment and the like;
-- `data info`：Data information, dataset name, input size and so on, such as imagenet, cifar, etc.;
+- `algorithm info`：包含算法名字的算法信息，例如simclr，mocov2等；
+- `module info`： 模块信息，用来表示一些 backbone，neck 和 head 信息；
+- `training info`：训练信息，即一些训练调度，包括批大小，学习率调度，数据增强等；
+- `data info`：数据信息：数据集名字，输入大小等，例如 imagenet，cifar 等。
 
-### Algorithm information
+### 算法信息
 ```
 {algorithm}-{misc}
 ```
 
-`Algorithm` means the abbreviation from the paper and its version. E.g:
-- `relative-loc` : The different word is concatenated by dashes `'-'`
+`Algorithm` 表示论文中的算法缩写和版本。例如：
+- `relative-loc`：不同单词之间使用破折线连接 `'-'`
 - `simclr`
 - `mocov2`
 
-`misc` offers some other algorithm related information. E.g.
+`misc` 提供一些其他算法相关信息。例如：
 - `npid-ensure-neg`
 - `deepcluster-sobel`
 
-### Module information
+### 模块信息
 ```
 {backbone setting}_{neck setting}_{head_setting}
```

Comment:
Please change this to `{backbone setting}-{neck setting}-{head_setting}`,  and the corresponding line in English version.